### PR TITLE
adding resend records

### PR DESCRIPTION
**Pull request by tobyab**
adding these records so we can send adults trying to join emails about... not joining!

### CODE REVIEW

**Review by Muirrum — CHANGES_REQUESTED** [review]
please set both TTLs to 600 per README and fix MX record test fail

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2046,6 +2050,15 @@ scrappy:
   ttl: 1
   type: CNAME
   value: glacial-dusk-446cvr5bsl9xqyp8yoge5tt7.herokudns.com.
+send:
+  - ttl: 1
+    type: MX
+    value: feedback-smtp.us-east-1.amazonses.com
+    priority: 10
+  - ttl: 1
```
Comment (alx-alexpark) [inline hackclub.com.yaml:2058]: Potential email spoofing risk?

File: hackclub.com.yaml [hunk 2]
```diff
@@ -2046,6 +2050,15 @@ scrappy:
...
+    type: TXT
+    value: v=spf1 include:amazonses.com ~all
```
Comment (alx-alexpark) [inline hackclub.com.yaml:2060]: Potential email spoofing risk?

**Comment by tobyab** [comment]
Not using Resend anymore! Appreciate you both for taking the time to review this though, @Muirrum and @alx-alexpark :)

**Pull request closed without merging**